earth_enterprise/BUILD_Ubuntu.md

```diff
@@ -35,10 +35,10 @@ sudo apt-get install \
     libgdbm-dev libgeos-dev libgeos++-dev libgif-dev libgtest-dev \
     libjpeg-dev libjpeg8-dev libmng-dev libogdi3.2-dev \
     libperl4-corelibs-perl libpng12-0 libpng12-dev libpq-dev libproj-dev \
-    libstdc++6 libtool libgif-dev libtiff-dev libgtk2.0-dev libglib2.0-dev \
+    libstdc++6 libtool libtiff-dev libgtk2.0-dev libglib2.0-dev \
```

Comment:
Do we need to add `libicu-dev` as well ?

Reply:
I don't know of a reason to add it. It's not a direct requirement of Open GEE that I'm aware of, and if any of these packages require it, apt should automatically bring it in.